### PR TITLE
Added ESW-0ZAA-EU device

### PR DESCRIPTION
**Pull request by dadavec**
It is rebranded Tuya TS004F device.

### CODE REVIEW

File: devices/moes.js [hunk 1]
```diff
@@ -151,4 +151,21 @@ module.exports = [
...
+        description: 'Wireless Scene Switch with 4 buttons',
+        exposes: [e.battery(), e.action(
+            ['on', 'off', 'brightness_move_up', 'brightness_step_up', 'brightness_step_down', 'brightness_move_down', 'brightness_stop'])],
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_move, fz.command_stop],
+        toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
+            try {
```
Comment (Koenkk) [inline devices/moes.js:167]: Can you try replacing this try catch with just `await reporting.batteryPercentageRemaining(endpoint);`, does the device configure correctly? (can be triggered via the frontend by clicking on the device -> yellow refresh button)

File: devices/tuya.js [hunk 2]
```diff
@@ -438,6 +438,7 @@ module.exports = [
         model: 'TS004F',
         vendor: 'TuYa',
         description: 'Wireless switch with 4 buttons',
+        whiteLabel: [{vendor: 'Moes', model: 'ESW-0ZAA-EU'}],
```
Comment (Koenkk) [inline devices/tuya.js:441]: Since this device has a separate definition in the moes.js file, this is not needed.

File: devices/moes.js (hunk 1 above)
Comment (dadavec) [reply devices/moes.js:167]: I just tried it and I am getting timeout error. But it behaves the same with the try/catch in place. I have found out, that when I click any button after configuration is started and before it times out. It seems to work fine. And it doesn't matter if try/catch is in place.

> Zigbee2MQTT:debug 2021-06-01 16:14:56: Received MQTT message on 'zigbee2mqtt/bridge/configure' with data 'Ctyrtlacitko'
Zigbee2MQTT:info  2021-06-01 16:14:56: Configuring 'Ctyrtlacitko'
Zigbee2MQTT:error 2021-06-01 16:15:06: Failed to configure 'Ctyrtlacitko', attempt 1 (Error: Bind 0x60a423fffed5b2d4/1 genPowerCfg from '0x00124b001cd40804/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/dist/utils/waitress.js:67:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7))


> Zigbee2MQTT:debug 2021-06-01 16:15:46: Received MQTT message on 'zigbee2mqtt/bridge/configure' with data 'Ctyrtlacitko'
Zigbee2MQTT:info  2021-06-01 16:15:46: Configuring 'Ctyrtlacitko'
Zigbee2MQTT:error 2021-06-01 16:15:56: Failed to configure 'Ctyrtlacitko', attempt 2 (Error: Bind 0x60a423fffed5b2d4/1 genPowerCfg from '0x00124b001cd40804/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/dist/utils/waitress.js:67:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7))

File: devices/tuya.js (hunk 2 above)
Comment (dadavec) [reply devices/tuya.js:441]: In such case, can you remove it from this pull request, or do I have to do it somehow?

**Comment by Koenkk** [comment]
I found out this exact device was already added in https://github.com/Koenkk/zigbee-herdsman-converters/pull/2624, this can be closed.

**Pull request closed without merging**